lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -74,25 +55,6 @@ class AmazonMarketplaceWebService < Service
       }
     }
 
-    ACTIONS = {
```

Comment:
At first glance it seems nice to have a list of all the endpoints we used, grouped. But when you're actually trying to understand the code and debug issues, you have constantly be scrolling back to the top of the file to figure out what `OPERATIONS[:outbound][:create]` actually points to. Just doesn't seem to buy us much.